modules/aws-acm-multi-zone/main.tf

```diff
@@ -0,0 +1,65 @@
+/**
+ * # AWS Generate multi zone cert
+ * Module for creating SSL certificate for multiple domains in different route53 zones (eg. different TLD domains).
+ * Limitation: Wildcard domains.
+ */
+# Partly based on https://gist.github.com/chancez/dfaaf799b98698839d65ebba55db7d44
+
+locals {
+  # Produces a list of maps of domain and there zone
```

Comment:
`# Produces a list of maps of domain and there zone` -> `# Produces a list of maps of domains and their zone`